packages/react/src/date-range-picker/DateRangePicker.tsx

```diff
@@ -20,15 +20,15 @@ import {
 } from '../core';
 import { FieldContainer, FieldHint, FieldLabel, FieldMessage } from '../field';
 import { visuallyHiddenStyles } from '../a11y';
-import { CalendarRange } from './Calendar';
-import { DateInput } from './DatePickerInput';
 import {
 	getValidDateRange,
 	parseDate,
 	formatDate,
 	constrainDate,
 	transformValuePropToInputValue,
-} from './utils';
+} from '../date-picker/utils';
+import { CalendarRange } from '../date-picker/Calendar';
+import { DateInput } from './../date-picker/DatePickerInput';
```

Comment:
Can these be imported from `../date-picker`?

I think if we're reaching into another component, we should not reach beyond the "public" API - ie the `index.ts`.

```suggestion
import { CalendarRange, DateInput } from '../date-picker';
```

Reply:
Ignore this. These  are not public api.